### PR TITLE
Updating HW information

### LINKED ISSUE #104: TEI notebook table is out of date

### PR DESCRIPTION
**Pull request by datavistics**
# What does this PR do?

Fixes #104  (issue)
We had some dated information in the table

## Who can review?

Feel free to tag members/contributors who may be interested in your PR.
@stevhliu

### CODE REVIEW

**Comment by merveenoyan** [comment]
@datavistics would you like to update the table instead? I actually liked it

**Comment by datavistics** [comment]
They have changed it a couple times. I was a bit worried it could go out of date again. Happy to add it if you think its worth it though. What do you think?

**Comment by stevhliu** [comment]
Maybe we can have just a few GPUs in the table as an example of what is offered and then tell users to follow the link for a more complete list? This way, you don't have to maintain this table in two places (Cookbook/Inference Endpoint docs) but users still get an idea of what is offered.

**Comment by datavistics** [comment]
I added most of them

File: notebooks/en/automatic_embedding_tei_inference_endpoints.ipynb [hunk 1]
```diff
@@ -113,18 +113,35 @@
    "id": "1e680f3d-4900-46cc-8b49-bb6ba3e27e2b",
```
Comment (stevhliu) [inline notebooks/en/automatic_embedding_tei_inference_endpoints.ipynb:113]: <p>"Inference Endpoints offers a number of GPUs that you can choose from."</p><p> </p><p>You can use a nice Tip callout here as well :)</p><p> </p><p>"&gt; [!TIP]</p><p> &gt; You may need to email us for access to some architectures."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/106/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

**Review by stevhliu — APPROVED** [review]
Thanks! I left a suggestion to make it less wordy, but otherwise, LGTM 🙂

**Comment by merveenoyan** [comment]
I agree with Steven's comments, once addressed we can merge IMO

**Comment by datavistics** [comment]
@stevhliu , I updated it, but Im not sure how to check if the tip formatting worked or not?

**Review by co42 — APPROVED** [review]
Thx !

**Comment by stevhliu** [comment]
I usually check through the doc-builder bot's [live preview](https://moon-ci-docs.huggingface.co/docs/cookbook/pr_106). The Tip formatting looks good, thanks again for the update!

**Pull request merged**